README.md

```diff
@@ -18,7 +18,7 @@ The detailed design of the `vm-memory` crate can be found [here](DESIGN.md).
 
 ### Platform Support
 
-- Arch: x86, AMD64, ARM64
+- Arch: x86, AMD64, ARM64, RISC-V
```

Comment:
Mh, sorry to both you about this, but this should probably be 
```suggestion
- Arch: x86_64, ARM64, RISCV64
```

(we don't support 32 bit, and all the other rust-vmm crates call it `x86_64` instead of AMD64)